styles/listenbrainz/catppuccin.user.css

```diff
@@ -0,0 +1,713 @@
+          color: transparent;
+          stroke: @mauve;
+        }
+      }
+    }
+
+    .card {
+      background-color: @surface0;
+      border-color: transparent;
+      box-shadow:
+        fade(@surface0, 10%) 0 1px 1px,
+        fade(@surface0, 15%) 0 2px 2px,
+        fade(@surface0, 20%) 0 4px 4px;
+
+      &#listen-count-card {
+        margin-bottom: 5px;
+        border-color: @surface1;
+      }
+
+      &.listen-card {
+        .listen-thumbnail > div {
+          color: @accent-color;
+          background-color: @surface1;
+          border-top-left-radius: 7px;
+          border-bottom-left-radius: 7px;
+        }
+
+        .listen-time {
+          color: @subtext0;
+        }
+        .listen-controls {
+          .btn.dropdown-toggle,
+          .btn.play-button,
+          .btn-transparent[title="Reset"] {
+            color: @text;
+            &:hover,
+            &.playing {
+              color: @accent-color;
+            }
+          }
+
+          .heart-actions();
+        }
+
+        &.playing-now,
+        &.current-listen {
+          background-color: @surface1 !important;
+
+          .listen-thumbnail > div {
+            background-color: @surface2;
+            height: 100%;
+          }
+        }
+      }
+    }
+
+    hr,
+    #listen-count-card hr,
+    .card-user-sn hr {
+      border-top-color: @surface1;
+    }
+
+    .follower-following-list,
+    .similar-users-list {
+      box-shadow: inset 0 11px 8px -10px @base;
+      > :not(:first-child) {
+        border-color: @surface1;
+      }
+    }
+
+    .progress {
+      background-color: @surface1;
+
+      .progress-bar.purple {
+        background-color: @mauve;
+      }
+
+      .progress-bar.orange {
+        background-color: @peach;
+      }
+
+      .progress-bar.red {
+        background-color: @red;
+      }
+    }
+
+    .dropdown-menu {
+      background-color: @overlay0;
+      button,
+      a {
+        color: @text !important;
+        &:visited {
+          color: @text !important;
+        }
+        &:hover {
+          background-color: fade(@accent-color, 30%) !important;
+        }
+      }
+    }
+
+    .pager li {
+      > a,
+      > a:focus,
+      > a:visited,
+      a:active,
+      > span {
+        background-color: @surface0;
+        border-color: @surface1;
+
+        &:hover {
+          background-color: @surface1;
+        }
+      }
+
+      > a,
+      > a:focus,
+      > a:visited,
+      > a:active {
+        color: @blue;
+        &:hover {
+          color: @sky;
+        }
+      }
+
+      &.disabled {
+        > a,
+        > a:focus,
+        > a:visited,
+        > a:active,
+        > span {
+          color: @text;
+          background-color: @surface0;
+          &:hover {
+            background-color: inherit;
+          }
+        }
+      }
+    }
+
+    .react-datetime-picker {
+      .react-datetime-picker__wrapper {
+        border-color: @mantle;
+      }
+      .react-calendar {
```

Comment:
The border I just didn't think to theme. The yellow day I didn't spot at all, since that's "today" when you have some other day selected. Both fixed. 👍